andela/mmdp_site
# MMDP-614: Center of invalidPage Component

**What does this PR do?**
- Fixes the bugs of page alignment

**description of Task to be completed?**

- Centers the invalid page component in the Resources page

**How should this be manually tested?**
- check out to this branch, `bg-center-Add-button-614`.
- start development server, `yarn dev`.
- Using the sidebar menu in your browser, click on the Resources.

**Any background context you want to provide?**

- None

**What are the relevant Jira issues?**

[MMDP-614](https://viisaus.atlassian.net/browse/MMDP-614)

**Screenshots (if appropriate)**

<img width="1680" alt="Screenshot 2019-06-06 at 10 33 02" src="https://user-images.githubusercontent.com/25298492/59014950-801d4180-8846-11e9-9ee8-8f2db16b6431.png">


**Questions:**

- None
